java/system-test/system-test-drivers/src/main/java/sleeper/systemtest/drivers/instance/AwsSleeperInstanceDriver.java

```diff
@@ -86,9 +87,9 @@ public boolean deployInstanceIfNotPresent(String instanceId, SleeperInstanceConf
             return false;
         }
         LOGGER.info("Deploying instance: {}", instanceId);
-        PopulateInstancePropertiesAws.builder(sts, regionProvider)
```

Comment:
The STS client and region provider fields here are no longer used. It looks like we will still need them though, once we make the necessary changes to UploadDockerImages and SyncJars.